### PR TITLE
add dr auto sync

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+### 灾难恢复
+
+当集群处于 `sync` 发生灾难，可满足 `RPO = 0` 恢复：
+
+- 主中心 PRIMARY 故障，丢失了大多数 Voter 副本，但是 DR 从中心有完整的数据，可从中心数据恢复。此时需要人工介入，通过专业工具恢复（恢复方式请联系 TiDB 团队）。
+- 从中心 DR 故障，丢失了少数 Voter 副本，能自动切换成 Async 模式。
+
+当集群不处于 `sync` 发生灾难，不能保证满足 `RPO = 0` 恢复：
+
+- 如果丢失了大多数 Vtoer 副本，需要人工介入，通过专业工具恢复（恢复方式请联系 TiDB 团队）。
```
Comment (disksing) [inline two-data-centers-in-one-city-deployment.md:231]:
```suggestion
- 如果丢失了大多数 Voter 副本，需要人工介入，通过专业工具恢复（恢复方式请联系 TiDB 团队）。
```

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+# # the deployments if a specific deployment value is missing.
+global:
+  user: "tidb"
+  ssh_port: 22
+  deploy_dir: "/data/tidb_cluster/tidb-deploy"
+  data_dir: "/data/tidb_cluster/tidb-data"
+
+server_configs:
+  tikv:
+    server.grpc-compression-type: gzip
```
Comment (disksing) [inline two-data-centers-in-one-city-deployment.md:54]: I think we can remove unrelated configs.

File: two-data-centers-in-one-city-deployment.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+  pd:
+    replication.location-labels:  ["zone","rack","host"]
+    schedule.tolerant-size-ratio: 20.0
```
Comment (disksing) [inline two-data-centers-in-one-city-deployment.md:57]: I think we can remove unrelated configs.

**Review by disksing — APPROVED** [review]

File: two-data-centers-in-one-city-deployment.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+wait-sync-timeout = "1m"
+```
+
+在上述配置文件中：
+
++ `replication-mode` 是启用设置复制模式，这里设置为 `dr-auto-sync`。默认情况为 majority。
++ `label-key` 用于区分不同的数据中心，需要和 placement rule 相匹配。其中主中心为 east，副中心为 west。
++ `primary-replicas` 在主中心 voter 副本的数量。
++ `dr-replicas` 是在副中心 voter 副本的数量。
++ `wait-store-timeout` 是当出现网络隔离或者故障是，切换到异步模式的等待时间。如果超过这个时间还没恢复则自动切换到异步复制模式。默认 60s。
```
Comment (TomShawn) [inline two-data-centers-in-one-city-deployment.md:197]: 前面没有提到异步模式和异步复制模式，请用语前后统一。
```suggestion
+ `dr-replicas` 是在从数据中心 voter 副本的数量。
+ `wait-store-timeout` 是当出现网络隔离或者故障时，切换到异步模式的等待时间。如果超过这个时间还没恢复则自动切换到异步复制模式。默认 60s。
```

File: two-data-centers-in-one-city-deployment.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+
+如果需要检查当前集群的复制状态，可以通过 API:
```
Comment (TomShawn) [inline two-data-centers-in-one-city-deployment.md:199]:
```suggestion
如果需要检查当前集群的复制状态，可以通过以下 API 获取：
```

File: two-data-centers-in-one-city-deployment.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+
+{{< copyable "shell-regular" >}}
+
+```bash
+% curl http://pd_ip:pd_port/pd/api/v1/replication_mode/status
```
Comment (TomShawn) [inline two-data-centers-in-one-city-deployment.md:204]:
```suggestion
curl http://pd_ip:pd_port/pd/api/v1/replication_mode/status
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**